Templates/TodoList-NodeJS/function.json

```diff
@@ -0,0 +1,33 @@
+﻿{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "httpTrigger",
```

Comment:
Since you only support GET and POST, recommend adding methods: [ "get", "post" ]. The runtime will then restrict.